src/Command/ProductCommand/DataLoadCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Command\ProductCommand;
+
+use Symfony\Component\Console\Command\Command;
+use Doctrine\Persistence\ManagerRegistry;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Output\NullOutput;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use App\Entity\Product;
+use App\Entity\ProductImage;
+
+/**
+ * Creates Product and ProductImage entities with relations and stores them in a database.
+ */
+class DataLoadCommand extends Command
+{
+    protected static $defaultName = 'data:load';
+    protected static $defaultDescription = 'Creates products and images with relations and stores them in a database.';
+    private ManagerRegistry $doctrine;
+
+    public function __construct(ManagerRegistry $doctrine)
+    {
+        parent::__construct();
+
+        $this->doctrine = $doctrine;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $command = $this->getApplication()->find('doctrine:fixtures:load');
```

Comment:
This is an interesting solution. The task was much simpler, you had to create a completely new command with $em->flush() , but running doctrine:fixtures:load also an option.